test/AutoDiff/compiler_crashers_fixed/issue-66522-pullback-generation-when-tangentvector-of-input-contains-nondifferentiable-fields.swift

```diff
@@ -0,0 +1,67 @@
+// RUN: %target-swift-frontend -emit-sil -O %s
```

Comment:
should we XFAIL it then?

Reply:
Ah, why? This does not cause a crash anymore.

Reply:
Does the comment at line 35 apply to the whole testcase? Or just the commented out lines?

Reply:
Just the commented out lines. Sorry, if that wasn't clear. 

I wanted to get Richard's comments on what seems like an edge case when using custom tangent vectors. But I can perhaps decouple it from this PR and handle it separately.

Reply:
+1